src/components/Hero.jsx

```diff
@@ -1,17 +1,131 @@
 import React from 'react'
+import styled from 'styled-components'
 
-const Hero = ({ eyebrowText, mainHeading, content, img, imgDescription }) => {
+const HeroSection = styled.section`
+  background-color: #b3ddf2;
+
+  @media (min-width: 769px) {
+    display: flex;
+    align-items: center;
+    padding: 0px 0px 0px 48px;
+    gap: 24px;
+  }
+`
+
+const ContentContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 16px;
+  padding: 48px 24px;
+
+  @media (min-width: 769px) {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 64px 0px;
+    gap: 24px;
+    width: 499px;
+    height: 330px;
+  }
+  @media (min-width: 1200px) {
+    width: 540px;
+  }
+`
+
+const EyebrowText = styled.span`
+  font-family: 'Source Sans Pro';
```

Comment:
We shouldn't need to specify the font anymore since it is added to the global styling.